docs(themeing): Move all themes in separate files (#1782)
IgniteUI/igniteui-docfx: en/components/themes/themes-list/bootstrap.md

### Switching from Material to Bootstrap theme

> Since version `9.0` our components support a new theme, built based on the [`Bootstrap 4`](https://getbootstrap.com/) and [`and angular bootstrap`](https://ng-bootstrap.github.io/#/home).  

Reviewer: ```suggestion
Since version `9.0` our components include a new theme inspired by [Bootstrap 4](https://getbootstrap.com/) and [ng-bootstrap](https://ng-bootstrap.github.io/#/home).
```

> In order to switch from `material` to `Bootstrap`, you can use one of the build in mixins.

Reviewer: ```suggestion
In order to switch from `Material` to `Bootstrap`, you can use one of the built-in mixins.
```

> For the `light` version of the `Bootstrap theme` you can use: 

Reviewer: ```suggestion
### Light Bootstrap Theme

For the `light` version of the Bootstrap theme you can use: 
```

```scss
@include igx-bootstrap-theme($bootstrap-palette);
```

> For the `dark` version of the `Bootstrap theme` you can use:

Reviewer: ```suggestion
### Dark Bootstrap Theme

For the `dark` version of the Bootstrap theme you can use:
```

```scss
@include igx-bootstrap-dark-theme($dark-bootstrap-palette);
```

> 

Reviewer: Please include links to the corresponding Sassdoc API documentation.
